com.unity.shadergraph/Editor/Resources/Styles/GraphSubWindow.uss

```diff
@@ -0,0 +1,127 @@
+﻿.unity-label {
+    padding-top: 5px;
+    padding-left: 2px;
+    padding-right: 2px;
+    padding-bottom: 2px;
+    margin-left: 4px;
+    margin-top: 2px;
+    margin-right: 4px;
+    margin-bottom: 2px;
+}
+
+.GraphSubWindow {
+    position:absolute;
+    border-left-width: 1px;
+    border-top-width: 1px;
+    border-right-width: 1px;
+    border-bottom-width: 1px;
+    border-radius: 5px;
+    flex-direction: column;
+    background-color: #2b2b2b;
+    border-color: #191919;
+    min-width: 100px;
+    min-height: 100px;
+    width: 200px;
+    height: 400px;
+}
+
+.GraphSubWindow.windowed {
+    position: relative;
+    padding-top: 0;
+    flex: auto;
+    border-left-width: 0;
+    border-top-width: 0;
+    border-right-width: 0;
+    border-bottom-width: 0;
+    border-radius: 0;
+    width: initial;
+    height: initial;
+}
+
+.GraphSubWindow.windowed > .resizer {
+    display: none;
+}
+
+.GraphSubWindow:selected {
+    border-color: #44C0FF;
+}
+
+.GraphSubWindow > .mainContainer {
+    flex-direction: column;
+    align-items: stretch;
+}
+
+.GraphSubWindow.scrollable > .mainContainer {
```

Comment:
I'm not great at USS/CSS, but I think these selectors can be simplified - see https://docs.unity3d.com/Manual/UIE-USS-WritingStyleSheets.html.